Tools/LyTestTools/ly_test_tools/_internal/pytest_plugin/editor_test.py

```diff
@@ -3,42 +3,49 @@
 For complete copyright and license terms please see the LICENSE at the root of this distribution.
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
-
-Utility for specifying an Editor test, supports seamless parallelization and/or batching of tests. This is not a set of
-tools to directly invoke, but a plugin with functions intended to be called by only the Pytest framework.
 """
+# Utility for supporting seamless parallel and/or batched sets of tests.
+# These tools are intended to be called only by the pytest framework and not invoked directly.
+
 from __future__ import annotations
-import pytest
+__test__ = False  # Avoid pytest collection & warnings since this module is for test functions, but not a test itself.
+
 import inspect
 
-__test__ = False
```

Comment:
this avoided the file to be caugh up if you do a pytest collection.
Filenames that that end with _test are picked up by the collector, if you add `__test__ = False`, pytest will ignore this file.

So we want to retain this line to avoid this to not be interpreted as a test. It is not a test, its an utility.